prymer/primer3/primer3.py

```diff
@@ -371,12 +365,25 @@ def design_primers(self, design_input: Primer3Input) -> Primer3Result:  # noqa:
                 f"Error, trying to use a subprocess that has already been "
                 f"terminated, return code {self._subprocess.returncode}"
             )
-
-        design_region: Span = self._create_design_region(
-            target_region=design_input.target,
-            max_amplicon_length=design_input.params.max_amplicon_length,
-            min_primer_length=design_input.params.min_primer_length,
-        )
+        design_region: Span
```

Comment:
I kept this in, @tfenne flagging this for you because I wasn't sure if we decided to take it out.